front/src/modules/spreadsheet-import/components/ModalWrapper.tsx

```diff
@@ -24,13 +27,15 @@ type Props = {
   children: React.ReactNode;
   isOpen: boolean;
   onClose: () => void;
+  size?: ModalSize;
```

Comment:
I don't think we should have a size and padding prop in spreadsheet-import. Let's just pass the right ones when instanciating the modal on line 38

Reply:
Yeah, totally makes sense. I don't know what I was thinking either. Must have been a result of coding while half asleep.